src/Uno.UWP/ApplicationModel/Calls/PhoneCallManager.Android.cs

```diff
@@ -20,16 +20,16 @@ static PhoneCallManager()
 					"PhoneCallManager was used too early in the application lifetime. " +
 					"Android app context needs to be available.");
 			}
-			_telephonyManager = (TelephonyManager)ContextHelper.Current
-				.GetSystemService(Context.TelephonyService);
+			_telephonyManager = (TelephonyManager)ContextHelper.Current!
```

Comment:
This may actually be an error. ContextHelper.Current must not be null here, and it would otherwise cause a NRE if used too early.

Reply:
@jeromelaban There is no way for the compiler to know if this is used early or not.

I actually noticed that I made `ContextHelper.Current` non-nullable and added the suppression inside the property getter so the suppression is no longer needed here.